modules/logging-content-filter-drop-records.adoc

```diff
@@ -10,7 +11,6 @@ When the `drop` filter is configured, the log collector evaluates log streams ac
 
 .Prerequisites
 
-* You have installed the {clo}.
 * You have administrator permissions.
 * You have created a `ClusterLogForwarder` custom resource (CR).
```

Comment:
This is not correct. You can create a clusterLogForwarder as the first time with all the needed and not needed to have it previously created.

What it's a requirement is to have create a serviceAccount to be used in the clusterLogForwarder linked to the correct clusterRoles depending on the log types allowed to this serviceAccount to collect. 

All the details are already mentioned in the PR https://github.com/openshift/openshift-docs/pull/80384/files exactly here https://github.com/openshift/openshift-docs/pull/80384/files#diff-7312d97cd1d4f7bf7f053f75108d26c4898afe1c9442033332be3b09358f75b5 and here https://github.com/openshift/openshift-docs/pull/80384/files#diff-8a01457b83c79be59c3ecdcc18d761f057790cab1de118070de9608994e11bf2

@libander